frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/common/AttachmentPortletUtils.java

```diff
@@ -71,12 +71,7 @@ public AttachmentPortletUtils(ThriftClients thriftClients) {
     private synchronized void makeConnector() throws TException {
         if (connector == null) {
             try {
-                AttachmentService.Iface client = thriftClients.makeAttachmentClient();
-                DatabaseAddress address = client.getDatabaseAddress();
-                connector = new AttachmentStreamConnector(address, downloadTimeout);
-            } catch (TException e) {
-                log.error("Could not get database address from attachment client...", e);
-                throw e;
+                connector = new AttachmentStreamConnector(downloadTimeout);
             } catch (MalformedURLException e) {
                 log.error("Invalid database address received...", e);
```

Comment:
The information is no longer present at this point. To log this debug information one would have to catch the exception earlier or pass the information.

We will have a look